mindsdb/integrations/handlers/postgres_handler/README.md

```diff
@@ -0,0 +1,58 @@
+# PostgreSQL Integration
+
+This documentation describes the integration of MindsDB with PostgreSQL, a powerful, open-source, object-relational database system. The integration allows for advanced SQL functionalities, extending PostgreSQL's capabilities with MindsDB's features.
+
+## Getting Started
+
+### Prerequisites
+
+   1. Ensure that MindsDB and PostgreSQL are installed on your system or you have access to cloud options.
+   2. If running locally install the dependencies as `pip install [postgres]`.
```

Comment:
This should be changed to `pip install mindsdb[postgres]`?

Reply:
Oh yes